bitcoincore/src/main/kotlin/io/horizontalsystems/bitcoincore/managers/BlockchainApi.kt

```diff
@@ -0,0 +1,57 @@
+package io.horizontalsystems.bitcoincore.managers
+
+import com.eclipsesource.json.Json
+import com.eclipsesource.json.JsonObject
+import io.horizontalsystems.bitcoincore.core.IInitialSyncApi
+import java.util.logging.Logger
+
+class BlockchainApi(host: String, authKey: String) : IInitialSyncApi {
```

Comment:
I copied `BCoinApi` and then added logic to fetch JWT.

Reply:
If you copied it can we delete the old class or does it need to stay?

Reply:
It needs to stay because LitecoinKit uses it